docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -44,8 +46,13 @@ However, actions in Play cannot be triggered by any external system, such as ext
 
 You have a few options to mock an external system:
 
-- In **Implement** mode, hard-code an example payload in the task or event **Output** section.
+- In **Implement** mode, hard-code an example payload in the task or event's **Example data** section in the properties panel.
 - When completing a task or event, use the secondary action to complete it with variables.
+- When filling forms or setting variables from Play, you can also save the variables to the BPMN file as example data to reuse them in future sessions.
+
+Play automatically uses example data from the BPMN file for many events and task types.
+If you want to use different data, you can override the example data by opening the secondary action menu on an element.
+The new data set will take precedent over the example data from the BPMN file for future Play sessions.
```

Comment:
this only happens if we save it though, right? it is possible to override the example data, but not save it, so it is only overridden for the existing play session.

(I might be wrong though, but this is how I remember it worked when testing the feature)

Reply:
@huygur Not quite, if you set data, this data will always be cached and used for the next session, too.